itests/testdata/issue_tx_testdata.go

```diff
@@ -0,0 +1,356 @@
+package testdata
+
+import (
+	"github.com/wavesplatform/gowaves/itests/config"
+	i "github.com/wavesplatform/gowaves/itests/fixtures"
+	utl "github.com/wavesplatform/gowaves/itests/utilities"
+	"github.com/wavesplatform/gowaves/pkg/proto"
+	"math/rand"
+	"time"
+)
+
+const (
+	letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789~!|#$%^&*()_+=\\\";:/?><|][{}"
+	errMsg      = "transactions does not exist"
+)
+
+type IssueTestData struct {
+	Account    config.AccountInfo
+	AssetName  string
+	AssetDesc  string
+	Quantity   uint64
+	Decimals   byte
+	Reissuable bool
+	Fee        uint64
+	Timestamp  uint64
+	ChainID    proto.Scheme
+	Expected   map[string]string
+}
+
+func NewIssueTestData(account config.AccountInfo, assetName string, assetDesc string, quantity uint64, decimals byte,
+	reissuable bool, fee uint64, timestamp uint64, chainID proto.Scheme, expected map[string]string) *IssueTestData {
+	return &IssueTestData{
+		Account:    account,
+		AssetName:  assetName,
+		AssetDesc:  assetDesc,
+		Quantity:   quantity,
+		Decimals:   decimals,
+		Reissuable: reissuable,
+		Fee:        fee,
+		Timestamp:  timestamp,
+		ChainID:    chainID,
+		Expected:   expected,
+	}
+}
+
+func RandStringBytes(n int) string {
+	b := make([]byte, n)
+	for j := range b {
+		b[j] = letterBytes[rand.Intn(len(letterBytes))]
+	}
+	return string(b)
+}
+
+func GetCurrentTimestampInMs() uint64 {
+	return uint64(time.Now().UnixNano() / 1000000)
+}
```

Comment:
You can use `time.Now().UnixMilli()`.
```suggestion
func GetCurrentTimestampInMs() uint64 {
	return uint64(time.Now().UnixMilli())
}
```

Reply:
Done